Gaijinizer patch/Map154.txt

```diff
@@ -134,9 +140,10 @@ Camazotz's happened. The rest is as you know
 カマソッソに拐われる時に、涙を浮かべながら
 必死に謝る彼女に、引き金を引けなかった……。
 > CONTEXT: Map154/events/4/pages/0/81/Dialogue/yuuma
-I...did not shoot her. With tears when being
-scolded by Camazotz. I couldn't trigger the girl
-who apologized desperately ....#MTLed
+I... didn't shoot her. I couldn't pull the
+trigger on that girl desperately apologizing
+with tears in her eyes as she was being carried
+off by Camazotz...#Ron
 > END STRING
```

Comment:
Minor wording and flow:
>_**I... couldn't shoot her. I couldn't pull the
trigger on that girl desperately apologizing
with tears in her eyes, as she was carried
off by Camazotz...**_